vars/runATH.groovy

```diff
@@ -0,0 +1,206 @@
+#!/usr/bin/env groovy
+
+/**
+ * Wrapper for running the ATH
+ */
+
+def call(Map params = [:]) {
+    def String athUrl = params.get('athUrl', 'https://github.com/jenkinsci/acceptance-test-harness.git')
+    def String athRevision = params.get('athRevision', 'master')
+    def String metadataFile = params.get('metadataFile', 'essentials.yml')
+    def String jenkins = params.get('jenkins', 'latest')
```

Comment:
Is this the latest LTS? master?

Reply:
latest weekly

Reply:
Maybe worth clarifying/documenting that. Thanks!